src/components/transaction-details.tsx

```diff
@@ -288,7 +288,34 @@ const transformDataToRowData = (tx: Transaction | MempoolTransaction, block?: Bl
         },
         children: tx.coinbase_payload.data,
       };
-      return [txid, sender, fees, nonce, blockTime, blockHash, scratch, canonical];
+      const altRecipient = {
+        condition: !!tx.coinbase_payload.alt_recipient,
+        label: {
+          children: 'Alt recipient',
+        },
+        children: tx.coinbase_payload.alt_recipient,
+      };
+      return [txid, sender, fees, nonce, blockTime, blockHash, scratch, canonical, altRecipient];
+    }
+    case 'smart_contract': {
+      const clarityVersion = {
+        condition: !!tx.smart_contract?.clarity_version,
```

Comment:
It would be correct and backward compatible to default to `clarity_version: 1` if the value isn't available.